dev/com.ibm.ws.beanvalidation.v20_fat/bnd.bnd

```diff
@@ -18,6 +18,7 @@ src: \
 	fat/src,\
 	test-applications/bvalApp/src,\
 	test-applications/bvalCDIApp/src,\
+	test-applications/bvalELApp/src,\
```

Comment:
Adds the new test application classes to the bnd source path so bnd will build it.